### PR TITLE
refactor: implement LERC20 to chainport existing v2 token (WIP)

### PR DESCRIPTION
**Pull request by Ignacio-Freire**

### CODE REVIEW

File: contracts/utils/ChainportLERC20.sol [hunk 1]
```diff
@@ -1,308 +1,1375 @@
...
+        require(_msgSender() == address(lossless), "LERC20: Only lossless contract");
+
+        uint256 fromLength = from.length;
+        uint256 totalAmount = 0;
+        
+        for(uint256 i = 0; i < fromLength;i++) {
+            address fromAddress = from[i];
+            require(confirmedBlacklist[fromAddress], "LERC20: Blacklist not confirmed");
+            uint256 fromBalance = super.balanceOf(fromAddress);
+            super.subBalance(fromAddress, fromBalance);
```
Comment (DomantasP) [inline contracts/utils/ChainportLERC20.sol:1331]: `super` not needed?

File: contracts/utils/ChainportLERC20.sol [hunk 2]
```diff
@@ -1,308 +1,1375 @@
...
+    event Mint(address indexed to, uint256 amount);
+    event AddressBlocked(address indexed addr);
+    event AddressUnblocked(address indexed addr);
+    event ChainportBridgeChanged(address indexed addr);
+    event ChainportCongressChanged(address indexed addr);
+    event StuckTokenWithdrawn(address indexed token, address indexed to, uint256 amount);
+    event BlackFundsDestroyed(address indexed token, uint256 amount);
+
+    /// Lossless Compliance
+    address public admin;
```
Comment (DomantasP) [inline contracts/utils/ChainportLERC20.sol:1089]: recoveryAdmin and related functionality is missing.

**Review by DomantasP — CHANGES_REQUESTED** [review]

File: contracts/utils/ChainportLERC20.sol (hunk 1 above)
Comment (Ignacio-Freire) [reply contracts/utils/ChainportLERC20.sol]: Removed on commit [22b8de3](https://github.com/Lossless-Cash/lossless-v3/pull/105/commits/22b8de34cd7a6c77e6169d07ceba8ffd23cd5e95)

File: contracts/utils/ChainportLERC20.sol (hunk 2 above)
Comment (Ignacio-Freire) [reply contracts/utils/ChainportLERC20.sol:1089]: Added on commit [a52c6cb](https://github.com/Lossless-Cash/lossless-v3/pull/105/commits/a52c6cb4c5e2dd25e1340e0d74add15e8bfa0ea9)

**Pull request merged**